src/Monomer/Widgets/Singles/DateField.hs

```diff
@@ -33,6 +33,7 @@ module Monomer.Widgets.Singles.DateField (
   dateField_,
   dateFieldV,
   dateFieldV_,
+  dateFieldD_,
```

Comment:
This change makes many of the `...D_` constructors public.

This is because the other constructors require a change event or a lens -- which you might not have if the user is never allowed to change the value.

On the other hand the same argument already applies to disabled widgets, so maybe there is a reason these constructors are not public. What do you think @fjvallarino ?

Reply:
> ...when in doubt, I prefer not to export things since changing them later may break user code.

I think this is sensible.

> To confirm: the idea of exposing, for example, textFieldD_ is to allow users to set a value without providing a change event? If that's the case, it makes total sense, yes.

Exactly that, yes.